html/projects.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
```

Comment:
P.S. I've had a couple beers tonight...  But why is a comment in the HTML referencing the github source of the JS needed?  Have you ever seen this in practice before?